src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -29,12 +29,19 @@ class LocalEngineFunctionsWithoutCallContext extends WdlStandardLibraryFunctions
   override protected def read_map(params: Seq[Try[WdlValue]]): Try[WdlMap] = {
     for {
       singleArgument <- extractSingleArgument(params)
-      if singleArgument.wdlType == WdlFileType
-      contents <- Success(Paths.get(singleArgument.asInstanceOf[WdlFile].valueString).slurp)
+      contents <- Success(fileContentsToString(singleArgument))
       wdlMap <- WdlMap.fromTsv(contents)
     } yield wdlMap
   }
 
+  override protected def read_object(params: Seq[Try[WdlValue]]): Try[WdlObject] = {
+    for {
+      singleArgument <- extractSingleArgument(params)
+      contents <- Success(fileContentsToString(singleArgument))
+      wdlObject <- WdlObject.fromTsv(contents)
+    } yield wdlObject
+  }
+
```

Comment:
Could be nice to implement `write_object()` too.  If not part of this PR, maybe file a ticket for implementing `write_object`, `read_objects`, and `write_objects`?